ceph-ansible-prs/config/definitions/ceph-ansible-prs.yml

```diff
@@ -0,0 +1,74 @@
+- project:
+    name: ceph-ansible-prs
+    scenario:
+      - ansible2.2-centos7-cluster
+      - ansible2.2-xenial-cluster
+    jobs:
+        - 'ansible-prs-{scenario}'
+
+
+- job-template:
+    name: 'ansible-prs-{scenario}'
+    node: vagrant&&ceph-ansible
+    concurrent: true
+    defaults: global
+    display-name: 'ceph-ansible: Pull Requests [{scenario}]'
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - github:
+          url: https://github.com/ceph/ceph-ansible
+    logrotate:
+      daysToKeep: 15
+      numToKeep: 30
+      artifactDaysToKeep: -1
+      artifactNumToKeep: -1
+
+    parameters:
+      - string:
+          name: sha1
+          description: "A pull request ID, like 'origin/pr/72/head'"
+
+    triggers:
+      - github-pull-request:
+          allow-whitelist-orgs-as-admins: true
+          org-list:
+            - ceph
+          trigger-phrase: 'jenkins test {scenario}'
+          only-trigger-phrase: false
+          github-hooks: true
+          permit-all: true
+          auto-close-on-fail: false
+          status-context: "Testing: {scenario}"
+          started-status: "Running: {scenario}"
+          success-status: "OK - {scenario}"
+          failure-status: "FAIL - {scenario}"
+
+    scm:
+      - git:
+          url: https://github.com/ceph/ceph-ansible.git
+          branches:
+            - ${sha1}
+          refspec: +refs/pull/*:refs/remotes/origin/pr/*
+          browser: auto
+          timeout: 20
+          skip-tag: true
+          wipe-workspace: false
+
+    builders:
+      - inject:
+          properties-content: |
+            SCENARIO={scenario}
+      - shell:
+          !include-raw:
```

Comment:
I think this needs to be ``!include-raw-escape`` now. Or does it not because you did ``$SCENARIO`` instead of ``{SCENARIO}`` in ``build``?